src/test/scala/battleship/gameActorSpec.scala

```diff
@@ -0,0 +1,100 @@
+package battleship
+
+/**
+  * Created by m06f947 on 3-8-2017.
+  */
+
+import akka.actor.{ActorRef, ActorSystem}
+import akka.testkit.{TestKit, TestProbe}
+import battleship.PlayerActor.{GetNextShot, PlaceBoats}
+import battleship.game.{BoardState, Boat, Game, LinearPlayer}
+import battleship.GameActor._
+import org.scalatest.{BeforeAndAfterAll, Matchers, WordSpecLike}
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+/**
+  * Created by jordidevos on 28/07/2017.
+  */
+class GameActorSpec(_system: ActorSystem)

```

Comment:
Class name doesn't correspond to filename (filename still starts with lowercase letter)